packages/client/src/generation/utils/buildDirname.ts

```diff
@@ -31,13 +31,13 @@ export function buildDirname(clientEngineType: ClientEngineType, relativeOutdir:
 function buildDirnameFind(relativeOutdir: string, runtimePath: string) {
   // potential client location on serverless envs
   const slsRelativeOutputDir = relativeOutdir.split(path.sep).slice(1).join(path.sep)
+  const slsRelativeOutputDirCode = slsRelativeOutputDir ? `\n${JSON.stringify(slsRelativeOutputDir)},` : ""
 
   return `
 const { findSync } = require('${runtimePath}')
 
 const dirname = findSync(process.cwd(), [
-    ${JSON.stringify(relativeOutdir)},
-    ${JSON.stringify(slsRelativeOutputDir)},
+    ${JSON.stringify(relativeOutdir)},${slsRelativeOutputDirCode}
```

Comment:
If `slsRelativeOutputDirCode` is empty, there will be a trailing comma :)

Reply:
trailing comma should be ok, and it was like that before